pydicom/sr/coding.py

```diff
@@ -0,0 +1,56 @@
+from collections import namedtuple
+
+from pydicom.dataset import Dataset
+from pydicom.sr._snomed_dict import mapping as snomed_mapping
+
+_CodeBase = namedtuple(
+    "Code", ("value", "scheme_designator", "meaning", "scheme_version")
+)
+_CodeBase.__new__.__defaults__ = (None,)
+
+
+class Code(_CodeBase):
+    """Namedtuple for representation of a coded concept consiting of the
```

Comment:
typo: consiting

Reply:
Still there...

Reply:
Ugh, I did change it - somehow  lost ... fixing and pushing again.